GithubHandler.py

```diff
@@ -25,6 +25,21 @@ def __init__(
         self.prs = self.repo.get_pulls(state="open", sort="created", base=main_branch)
         self.prs_nums = [pr.number for pr in self.prs]
         self.prs_dict = {k: v for k, v in zip(self.prs_nums, self.prs)}
+        self.accepted_file_extensions = [
+            ".py",
```

Comment:
Defining allowed file extensions is helpful. Ensure this list is consistent with the project standards and add any missing file types relevant to the project.
        ```suggestion
        self.accepted_file_extensions = [
            ".py",
             ...
            ".jsx",
        ]
        ```